.github/actions/setup-python/action.yml

```diff
@@ -24,10 +24,10 @@ runs:
           python_version=$(cat .python-version)
         fi
         if [[ "$python_version" == "3.10" ]]; then
-          if [ ${{ runner.os }} == "Windows" ]; then
-            python_version="3.10.11"
-          else
+          if [ ${{ runner.os }} == "Linux" ]; then
             python_version="3.10.16"
+          else
+            python_version="3.10.11"
```

Comment:
This PR adds a new lint jov for macos. The setup python action doesn't provide Python 3.10.16 for arm64 macos.